packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -698,12 +737,25 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
     // Are we at a new index? If so, update the state, otherwise
     // there is nothing to do
-    if (index !== selectedIndex) {
+    if (this.props.multiSelect || selectedIndices.length < 1 || (selectedIndices.length === 1 && selectedIndices[0] !== index)) {
```

Comment:
Is onChanged (below, line 769) only called with the newest value added?

Reply:
onChanged is called each time a checkbox toggles.